src/bench.rs

```diff
@@ -0,0 +1,76 @@
+use std::path::Path;
+
+use clap::ArgMatches;
+use tokio::fs;
+
+use crate::{
+    error::AocError,
+    util::{file::*, get_day},
+};
+
+async fn create_bench_foler(path: &Path) -> Result<(), AocError>
+{
+    fs::create_dir(path.join(".bench")).await?;
+    fs::create_dir(path.join(".bench").join("bench")).await?;
+
+    // It can probably _just_ live in the .bench folder? test
+    let folder = path.join(".bench").join("bench");
+
+    let template = format!("{}/template/Cargo.toml.benchmark", env!("CARGO_MANIFEST_DIR"));
+
+    fs::copy(template, folder.join("Cargo.toml")).await?;
```

Comment:
I think we can use `create_dir_all` to avoid repeated calls to `create_dir`(?)

Reply:
I think I clicked the wrong line to comment on, but I think you understand what I meant😅